### DISCUSSION TITLE
Does doks support multiple areas?

### DISCUSSION THREAD

**Discussion by LunaticMuch** [thuliteio/doks#157] [Q&A]
I am trying to figure out if doks support multiple areas for documentations. Say that I want to have a `/docs` section for my docs and another one `/external-resources` for external docs, which I further slice into more sections. Is this possible?

**Comment by h-enk** [comment]
Absolutely. Just set it up like the docs tree — making sure to add unique menus.

See the showcase tree on the Doks website for an example.

**Reply by LunaticMuch** [reply]
Got it. I am duplicating the layout for the section, it's preferable to me than adding the layout to the frontmatter.
